api/nakadi-event-bus-api.yaml

```diff
@@ -2328,6 +2386,62 @@ definitions:
         description: |
           Number of milliseconds that Nakadi stores events published to this event type.
 
+  EventTypeAuthorization:
+    type: object
+    description: |
+      Authorization section for an event type. This section defines three access control lists: one for producing
+      events ('write'), one for consuming events ('read'), and one for administering an event type ('admin').
+      Regardless of the values of the authorization properties, administrator accounts will always be authorized.
+    properties:
+      admin:
+        type: array
+        description: |
+          An array of subject attributes that are required for updating the event type. Any one of the attributes
```

Comment:
In the literature on access control, attributes are categorised in `subject attributes`, `action attributes`, `resource attributes`, and `environment attributes` (see e.g. the XACML standard: http://docs.oasis-open.org/xacml/3.0/xacml-3.0-core-spec-os-en.html#_Toc325047262). In this case we try to keep things as simple as possible, so we do not want to write complex rules. In particular, attributes will always be subject attributes.

But, if it is less confusing to simply call them `attributes` or `authorization attributes`, then I think we should go with the least confusing option.